ideas/dropdown.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+// example for one button
+
+function handleNavClick() {
+	$('.js-dropdown-btn-classes').click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-classes').toggleClass('show');
+	});
+
+	window.onclick = function(e) {
+		if (!e.target.matches('.js-dropdown-btn-classes')) {
+			$('.js-actions-dropdown-classes').removeClass('show');
+		}
+	}
+}
+
+function handleMoreClick() {
+	$('.js-dropdown-btn-1').click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-1').toggleClass('show');
+	});
+
+	window.onclick = function(e) {
```

Comment:
Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu